### ISSUE TITLE
Husky trying to install when consuming this package.

### ISSUE THREAD

**Issue by msluther** [wojtekmaj/enzyme-adapter-utils#2]
https://github.com/wojtekmaj/enzyme-adapter-utils/blob/66d57ed01f1dce29708a7f663f22d452b09eed31/package.json#L11

I believe that should have been a `prepare` instead of `postinstall`. The latter will run whenever someone is installing this package, which doesn't seem like what you wanted here.  ref: https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts

**Comment by msluther** [comment]
I was getting the following when trying a fresh install of the react-17 enzyme adapter:

```
npm ERR! code 127
npm ERR! path /Users/me/my-repo/node_modules/@wojtekmaj/enzyme-adapter-utils
npm ERR! command failed
npm ERR! command sh -c husky install
npm ERR! sh: husky: command not found
```

**Comment by msluther** [comment]
As a workaround pinning to the previous version in your project seems to work:

```json
"@wojtekmaj/enzyme-adapter-utils": "0.1.2",
```

**Comment by steezeburger** [comment]
Also experiencing this issue. It has broken our ci/cd.
Pinning in our project does not work, as we aren't the ones using this, but a child dependency. So `husky install` is still ran and failing.

Edit: `yarn install` does not work, but `npm install` appears to work with the suggested version pinning.

**Comment by slutske22** [comment]
Having this issue as well, totally crashing our `npm install` script

**Comment by johnny-mcfadden-dailypay** [comment]
Very happy to see others having this issue, was stumped for a while trying all sorts. Locking the version `"@wojtekmaj/enzyme-adapter-utils": "0.1.2",` as @msluther said seemed to work okay for npm install.

**Comment by pmconne** [comment]
0.1.4 resolves the problem for me - thanks @wojtekmaj!

**Comment by wojtekmaj** [comment]
Yep sorry about that. Fixed in c30e4250704ceff1b779789e5a183582ba18952d.
